include/variables.h

```diff
@@ -1268,7 +1268,7 @@ extern UNK_PTR D_801C02F8;
 // extern UNK_TYPE1 D_801C07F0;
 // extern UNK_TYPE1 D_801C0820;
 // extern UNK_TYPE1 D_801C0838;
-// extern UNK_TYPE1 D_801C0850;
+extern Gfx D_801C0850;
```

Comment:
Display lists (`Gfx`s) always are arrays (or pointers).
```suggestion
extern Gfx D_801C0850[];
```